src/test/java/org/prebid/server/bidder/yahoossp/YahooSSPBidderTest.java

```diff
@@ -44,14 +55,23 @@ public class YahooSSPBidderTest extends VertxTest {
 
     private YahooSSPBidder yahooSSPBidder;
 
+    @Rule
+    public final MockitoRule mockitoRule = MockitoJUnit.rule();
+
+    @Mock
+    private BidRequestOrtbVersionConversionManager conversionManager;
+
```

Comment:
`@Rule` and `@Mock` fields should be top fields of the class( before yahooSSPBidder )